tests/release/release.go

```diff
@@ -0,0 +1,133 @@
+package release
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/google/uuid"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/redhat-appstudio/e2e-tests/pkg/framework"
+	gitopsv1alpha1 "github.com/redhat-appstudio/managed-gitops/appstudio-shared/apis/appstudio.redhat.com/v1alpha1"
+
+	// appstudiov1alpha1 "github.com/redhat-appstudio/release-service/api/v1alpha1"
```

Comment:
Let's remove unused imports, not leave it commented

Reply:
Fixed.